tests/Application/Entity/OrderLogEntry.php

```diff
@@ -13,42 +13,11 @@
  */
 class OrderLogEntry extends LogEntry
 {
-// We don't need ID here
-// As it should be inherited from LogEntry
-// But for some reason mapping of LogEntry not inheriting...
-// And if we will see d:schema:update --dump-sql command -
-// no LogEntry fields there that is weird...
-//    /**
-//     * @var int|null
-//     *
-//     * @ORM\Id
-//     * @ORM\GeneratedValue
-//     * @ORM\Column(type="integer")
-//     */
-//    protected $id;
-
     /**
      * @var Order|null
      *
      * @ORM\ManyToOne(targetEntity="Tests\Setono\SyliusLogEntryPlugin\Application\Entity\Order", inversedBy="logEntries")
      * @ORM\JoinColumn(name="owner_id", nullable=false, onDelete="CASCADE")
      */
     protected $owner;
-
```

Comment:
This is example of how things can be overriden at app. So I would like to keep this.